lte/gateway/docker/docker-compose.yaml

```diff
@@ -39,7 +39,7 @@ x-agw-c-service: &cservice
 # Generic anchor for go services
 x-agw-go-service: &goservice
```

Comment:
Does `x-agw-go-service` make sense a as name when it defines an FEG image?

Reply:
related: Does it even make sense to have an indirection here that is just used once? we could just let the envoy_controller reference `*service` and define the image. No need for an intermediate thing, and thus no need for a naming discussion ;)

Reply:
Valid. Changed. :heavy_check_mark: